packages/pancake-svg/src/svg.js

```diff
@@ -0,0 +1,122 @@
+/***************************************************************************************************************************************************************
+ *
+ * Generate and compile SVG's
+ *
+ * @repo    - https://github.com/govau/pancake
+ * @author  - Dominik Wilkowski
+ * @contributor - Michael Arthur
+ * @license - https://raw.githubusercontent.com/govau/pancake/master/LICENSE (MIT)
+ *
+ **************************************************************************************************************************************************************/
+
+
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+// Dependencies
+//--------------------------------------------------------------------------------------------------------------------------------------------------------------
+import Path from 'path';
+import Fs from 'fs';
+import SVGSpriter from 'svg-sprite';
+const svgToImg = require('svg-to-img');
```

Comment:
I would stay with the same `import` notation

Reply:
Yeah that package wasn't happy about import/the way it got trans-piled did not work with .default.func, will track down the proper way to deal with that.

Reply:
Seems to be exported as a constant `from` ->  https://github.com/etienne-martin/svg-to-img/blob/master/dist/index.js#L108 😕 

Try: `import { from } from 'svg-to-img'` ?

Reply:
Yeah I figured it would be that! thanks.